manager/manager.go

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright 2020 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package manager
+
+import (
+	"context"
+	"crypto/tls"
+	"net/http"
+
+	"github.com/go-kit/kit/endpoint"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	kithttp "github.com/go-kit/kit/transport/http"
+	"github.com/gorilla/mux"
+	argus "github.com/xmidt-org/argus/model"
+	"github.com/xmidt-org/mimisbrunnr/dispatch"
+	"github.com/xmidt-org/mimisbrunnr/model"
+	"github.com/xmidt-org/mimisbrunnr/norn"
+	"github.com/xmidt-org/webpa-common/logging"
+	"github.com/xmidt-org/wrp-go/v2"
+)
+
+type Manager struct {
+	nornsDispatch    map[string]nornDispatcher
+	dispatcherConfig dispatch.DispatcherConfig
+	logger           log.Logger
+}
+
+type nornDispatcher struct {
+	norn       model.Norn
+	dispatcher dispatch.D
+}
+
+func NewManager(dc dispatch.DispatcherConfig, logger log.Logger) (*Manager, error) {
+	return &Manager{
+		dispatcherConfig: dc,
```

Comment:
```suggestion
		nornsDispatch: map[string]nornDispatcher{},
		dispatcherConfig: dc,
```
If we don't make a map, the value will be nil, which will cause problems later.